frontend/packages/knative-plugin/src/utils/condition-utils.ts

```diff
@@ -0,0 +1,13 @@
+import * as _ from 'lodash';
+import { K8sResourceCondition, K8sResourceConditionStatus } from '@console/internal/module/k8s';
+
+export const getConditionOKCount = (conditions: K8sResourceCondition<any>[]): number =>
+  _.sumBy<any>(conditions, (c) => Number(c.status === K8sResourceConditionStatus.True));
+
+export const getConditionString = (conditions: K8sResourceCondition<any>[]): string =>
+  `${getConditionOKCount(conditions)} OK / ${_.size(conditions)}`;
+
+export const getCondition = <T extends any>(
+  conditions: K8sResourceCondition<any>[],
+  type: string,
+): K8sResourceCondition<T> | undefined => _.find(conditions, (c) => c.type === type);
```

Comment:
Updated a line to simplify the typing. However `any` type is still required in these because they are meant to work with any type of `K8sResourceCondition`.